src/components/accordian/Accordian.tsx

```diff
@@ -17,15 +18,19 @@ type TProps = {
 
 export const Accordian = ({
```

Comment:
Thought for later: It'd be nice to replace this entirely with Base UI's accordion when we get some time.